.travis.yml

```diff
@@ -8,8 +8,11 @@ python:
     - nightly
 
 before_install:
-    - pip install --upgrade setuptools
-    - pip uninstall -y six
+  # clean existing dependencies to avoid clashes
+  - pip freeze | xargs pip uninstall -y
+  - pip uninstall -y setuptools
+  - pip install setuptools==33.1.1  # pin specific version in virtualenv
```

Comment:
Why install? Wouldn't it work to only uninstall? Buildout should install a new version.

Reply:
@mxm No it does not. It would fail with `ModuleNotFoundError: No module named 'pkg_resources'` in Python 3.6 or newer